### DISCUSSION TITLE
How to increase the width of file/dir section in preview pane?

### DISCUSSION THREAD

**Discussion by LinArcX** [leo-arch/clifm#277] [Q&A] [answered]
As you can see, the left section that contains the names of files/directory is too narrow than i can't read the names. I think it's related to fzf settings, but not sure:
![2024-03-31_20-31](https://github.com/leo-arch/clifm/assets/10884422/9998010a-e43c-40be-b3e4-829815a024dc)
Anyone has some ideas to fix this issue?

Edit: These are *FZF* related values in env:(windows not msys2, i didn't configure fzf on my msys2 yet.)
```
FZF_DEFAULT_COMMAND=rg --files --no-ignore --hidden --follow --glob "!.git/*"
FZF_CTRL_U_COMMAND=ls -ltrah --color=auto
FZF_COMPLETION_AUTO_COMMON_PREFIX=true
FZF_DEFAULT_OPTS=--layout=reverse --border --cycle
```

OS: windows 10, msys2
clifm version: compiled directly from git

**Comment by leo-arch** [comment, marked as answer]
> I think it's related to fzf settings, but not sure

This is most likely the case. Try unsetting FZF environment variables for **clifm**

```sh
FZF_DEFAULT_COMMAND="" FZF_CTRL_U_COMMAND="" FZF_COMPLETION_AUTO_COMMON_PREFIX="" FZF_DEFAULT_OPTS="" ./clifm
```

By default, **clifm** passes a set of options to `fzf`, which might conflict with some of your environment variables. To take a look at these default options, run `cs edit` and browse to the bottom of the file: `FzfTabOptions` is what you're looking for.

EDIT: According to my tests, the problem is `--border`. Try removing just this option from your environment. It seems to be getting in the way of **clifm**'s internal file name length calculation.

EDIT 2: Or, you can just add `--border=none` to `FzfTabOptions`.

**Reply by LinArcX** [reply]
> Or, you can just add --border=none to FzfTabOptions

Thanks a lot.

**Reply by leo-arch** [reply]
With the latest commit, the `--border` option should no longer be an issue (i.e. you should be able to remove `--border=none` from your `FzfTabOptions` line without causing any problem).

**Reply by LinArcX** [reply]
Thanks. Beside all these things, is it possible to increase the width or height of the preview window?(each side separately and the whole pane)

**Reply by leo-arch** [reply]
Maybe to a fixed value. What are you trying to achieve exactly?

EDIT: You can give the [view command](https://github.com/leo-arch/clifm/wiki/Introduction#view) a try. It's basically a full-screen files previewer (for the current directory only) based on `fzf`. Or maybe the `fzfnav` plugin, which can be invoked via the `-` command (yes, just a dash).
